docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
     - couchdb
 
   ldap_proxy:
-    image: benel/aaaforrest
+    image: benel/aaaforrest:latest
```

Comment:
Isn't it the same?

Reply:
According to my experience, the image did not update with `docker-compose pull`. It did after I add `:latest`. 

It also surprised me. However, perhaps it do/should work